plutus-ledger-api/src/v1/value.rs

```diff
@@ -105,6 +318,16 @@ impl TokenName {
     }
 }
 
+impl TokenName {
+    pub fn from_bytes(bytes: Vec<u8>) -> Self {
+        TokenName(LedgerBytes(bytes))
```

Comment:
Hmmm - `from_string` below implies that a token name stores some utf-8 encoded bytes, should we also validate here? Or perhaps it'd be a great idea to introduce a newtype instead?

Reply:
I only added a `from_string` and not a `to_string` (which might be a future addition)
`from_string` is safe

Reply:
Now I also added `try_to_string` with returns a Result